nextflow.config

```diff
@@ -24,39 +157,55 @@ params {
   plaintext_email = false
   monochrome_logs = false
   help = false
-  igenomes_base = 's3://ngi-igenomes/igenomes/'
+//  igenomes_base = 's3://ngi-igenomes/igenomes/'                                                   // pipeline does not support igenomes
   tracedir = "${params.outdir}/pipeline_info"
-  igenomes_ignore = false
-  custom_config_version = 'master'
-  custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
-  hostnames = false
-  config_profile_description = false
-  config_profile_contact = false
-  config_profile_url = false
-
+//  igenomes_ignore = false                                                                         // pipeline does not support igenomes
   // Defaults only, expecting to be overwritten
   max_memory = 128.GB
   max_cpus = 16
   max_time = 240.h
 
 }
 
-// Container slug. Stable releases should specify release tag!
-// Developmental code should specify :dev
+
+//--------
+// Work directory:
+//--------
+workDir = "$baseDir/work"
+
+
+//--------
+// Container:
+//--------
 process.container = 'nfcore/dualrnaseq:dev'
 
 // Load base.config by default for all pipelines
 includeConfig 'conf/base.config'
 
+
 // Load nf-core custom profiles from different Institutions
 try {
   includeConfig "${params.custom_config_base}/nfcore_custom.config"
 } catch (Exception e) {
   System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/nfcore_custom.config")
 }
 
+// Load nf-core/dualrnaseq custom profiles from different Institutions
+try {
+  includeConfig "${params.custom_config_base}/pipeline/dualrnaseq.config"
+} catch (Exception e) {
+  System.err.println("WARNING: Could not load nf-core/config/dualrnaseq profiles: ${params.custom_config_base}/pipeline/dualrnaseq.config")
+}
+
+// to read your own custom genome file named computing_platform.config and saved in conf folder. 
```

Comment:
Custom genome or computing platform?

What will this file will be used for

Reply:
Another good spot. Kinda linked to the comment above.
On our institute cluster, we use a SGE and have some basic platform-based settings here, the settings are stored in `computing_platform.config`.
I guess a good work around would be to have this file available with detailed comments within so it provides an example for others. 
But I'll leave this line (`//includeConfig 'conf/computing_platform.config' `) as a comment so as to not confuse when running initially.
Would you agree?

Reply:
SGE, that would be the executor then, right?
What else would be in this config file?

Not sure where you're going with that

Reply:
The computing_platform.config contains all information necessary to run the pipeline on our institutional cluster. We will keep these parameters in the pipeline-specific config file. However, I wonder if there is a possibility to simplify for potential users from our institute to define the user-specific parameters, such as singularity.cacheDir, workDir, and executor.queueSize, so they can know that this information has to be specified if they want to run the pipeline on the cluster.     

These are the information we keep in the computing platform.config file:
		executor.name = 'sge'
		executor.queueSize = 6
		executor.jobName = "dual_rna_seq"
		process.clusterOptions='-l arch=linux-x64 -b n -q all.q -e /vol/projects/bmikagos/Dual_RNA_seq/logs/ -o /vol/projects/bmikagos/Dual_RNA_seq/logs/'
		process.penv = 'multislot'
                singularity.cacheDir = "/vol/cluster-data/bmikagos/"
		workDir = "/vol/projects/bmikagos/Dual_RNA_seq/work"

Reply:
ok, then I would advise about not putting any line about computing platform at all, this is all configs, you can point to the nf-core/configs repo for that.